Tasks/AzureStaticWebAppV0/task.json

```diff
@@ -14,8 +14,8 @@
   "demands": [],
   "version": {
     "Major": "0",
-    "Minor": "194",
-    "Patch": "2"
+    "Minor": "198",
```

Comment:
Could you please bump the task version up to 200?

Reply:
@alexander-smolyakov All done